### DISCUSSION TITLE
Some issues for first-time development using Uno

### DISCUSSION THREAD

**Discussion by shingming** [unoplatform/uno#8908] [Q&A] [answered]
Hi, I am Xamarin.forms/MAUI developer and recently want to try UNO platform.

I use JetBrains Rider and follow the uno starting guides. I install gtk+3  on Mac using the following command line:
`brew install gtk+3`
![Screenshot 2022-05-28 at 3 30 48 PM](https://user-images.githubusercontent.com/65477270/170818503-08444f3f-fde3-4e12-bc73-ff7f29f7c22e.png)

_issue 1_
After installation, I run the uno checker `uno-check`. But, there is no information about gtk+3 from the output info.
![Screenshot 2022-05-28 at 3 30 17 PM](https://user-images.githubusercontent.com/65477270/170818514-4d003a13-1234-48fd-b6e4-52e6e28f4f5d.png)

_issue 2_
I create a new uno project using the following command line:
`dotnet new unoapp -o MyApp`

I will get a warning and error message. It seems like the macOS system reason (can't create WPF project). If I use macOS for development, is it normal to get the following error or warning?
<img width="1512" alt="Screenshot 2022-05-28 at 3 46 55 PM" src="https://user-images.githubusercontent.com/65477270/170818713-c4279b0f-4ea3-4301-b583-b0112887145c.png">

**Comment by jeromelaban** [comment, marked as answer]
Thanks for the report. For the first point, this is expected, there's no specific message showing up when GTK3 is installed properly.

For the second point, this is also expected. Building for WPF is not supported on mac, but custom selection is not possible using the default `dotnet new` templates. You can remove the WPF target when creating the app:

```
dotnet new unoapp -skia-wpf=false
```

See [here](https://platform.uno/docs/articles/get-started-vscode.html?tabs=windows) for more details.

**Reply by shingming** [reply]
Thank you : )
